sticky_notes.py

```diff
@@ -81,3 +81,26 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+
+    @cmd
+    def del_note(self, event):
+        args = event.args
+        from_nick = event.meta['nick']
+
+        if len(args) < 1:
+            return "Missing nick argument."
+
+        if len(args) == 1:
```

Comment:
So if I use more than 1 argument to this command that will not be recognized at all?